### PR TITLE
bugfix/update-fasta-help

### PR DESCRIPTION
**Pull request by kopichris**
This PR is related to issue #491.

The primary purpose of this pull request is to update the help text for the `--fasta` parameter.

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] `CHANGELOG.md` is updated.

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -261,8 +261,7 @@
                 },
                 "save_merged": {
                     "type": "boolean",
-                    "description": "Save merged reads.",
-                    "default": false
```
Comment (atrigila) [inline nextflow_schema.json:265]: Why is the default false removed here?

Comment (kopichris) [reply nextflow_schema.json:265]: I'll update the PR.  Not sure why this got removed.  Thanks for catching this.

Comment (kopichris) [reply nextflow_schema.json]: Updated the PR in https://github.com/nf-core/smrnaseq/pull/494/commits/d7688338b0d38b5ec60e811b8862d49a6d191f18.  Thanks again!

**Review by atrigila — APPROVED** [review]
Looks good, thank you for your contribution!

**Pull request merged**